uk.co.compucorp.civicrm.hrleaveandabsences/tests/phpunit/CRM/HRLeaveAndAbsences/BAO/LeaveRequestDateTest.php

```diff
@@ -92,31 +86,32 @@ public function testGetDatesForLeaveRequestReturnsEmptyArrayForANonExistingLeave
   }
 
   public function testDeleteDatesDeletesTheDatesForASpecificLeaveRequest() {
-    LeaveRequestDate::create([
-      'date' => date('YmdHis'),
-      'leave_request_id' => 1,
-    ]);
-
-    LeaveRequestDate::create([
-      'date' => date('YmdHis'),
-      'leave_request_id' => 2,
+    $leaveRequest1 = LeaveRequestFabricator::fabricateWithoutValidation([
```

Comment:
you should not need the LeaveRequest Fabricator here as well (see my comment about dependencies on the previous test)